node/runtime/src/impls.rs

```diff
@@ -20,8 +20,16 @@ use node_primitives::Balance;
 use runtime_primitives::weights::{Weight, WeightMultiplier};
 use runtime_primitives::traits::{Convert, Saturating};
 use runtime_primitives::Fixed64;
-use support::traits::Get;
-use crate::{Balances, MaximumBlockWeight};
+use support::traits::{Get, OnUnbalanced, Currency};
+use crate::{Balances, Authorship, MaximumBlockWeight, NegativeImbalance};
+use crate::constants::fee::TARGET_BLOCK_FULLNESS;
+
+pub struct Author;
```

Comment:
As promised, I moved some other structs that implement stuff from the top level `lib.rs` here. Same rationale as `constants.rs`: trying to keep the `lib.rs` file concise and tidy.